packages/picasso.js/src/core/scales/ticks/__tests__/tick-generators.spec.js

```diff
@@ -205,11 +205,12 @@ describe('Tick generators', () => {
         scale.domain([-99, 99]);
         const ticks = scale.ticks(input);
         const majorTicks = ticks.filter((t) => !t.isMinor);
+
         expect(majorTicks[0]).to.deep.equal({
           position: 0,
           start: 0,
           end: 0,
-          label: '-99',
+          label: '−99',
```

Comment:
[d3-format](https://observablehq.com/@d3/d3-format#cell-107) changed the minus sign to `U+2212`